File: Backend/src/main/java/com/app/distrifoods/app/entities/Cliente.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.app.distrifoods.app.entities;

import java.io.Serializable;
import javax.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Entity
@Table(name="clientes")
public class Cliente implements Serializable{
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    @Column(name="id_cliente")
    private Integer id;
    @Column(name="id_usuario", unique=true)
    private Integer idUsuario;
    @Column(nullable=false)
    private String direccion;
    @Column(nullable=false)
    private Long telefono;
}

File: Frontend/rotonda-front/src/app/models/cliente.ts
import { Usuario } from "./usuario";

export interface Cliente {
  id:  number;
  idUsuario: number;
  direccion: string;
  telefono: number;
}


export interface ClienteCompleto {
  id:  number;
  usuario: Usuario;
  direccion: string;
  telefono: number;
}
